content/release-10-18-0/whatsnew-10-18-0.md

```diff
@@ -7,6 +7,39 @@ layout: bundle
 Release 10.18.0 includes the following new features or major feature enhancements.
 
 ### Application enablement
+#### Global Time Selector for dashboards
+
+A new global time context feature has been introduced, allowing users to apply a consistent time context (e.g., display data from the last hour, last month, etc.) across an entire dashboard. Widget developers can now set the global time context for the whole dashboard or a particular widget. Widgets bound to the global time context are marked accordingly. Users can also share dashboards with a specific time context by including the time context parameters in a URL.
+
+The global time context is available for the following existing widgets: Event list, Data points graph, and Data points table.
+
+#### Map feature migration and enhancements
+
+The map feature has been migrated from AngularJS to Angular, bringing along several improvements and modifications:
+
+Breaking changes:
+* Real-time tracking has been limited to a single device at a time.
+  
+New features:
+
+* A device can now be followed in real-time.
+* Device icons can be customized.
+* Zoom level adjustments have been introduced.
+* Center point settings have been added.
+* Clustering support has been implemented to handle a high number of devices.
+* Devices are now loaded exclusively within the visible map area.
+* Refresh polling capabilities have been incorporated for multiple devices, allowing users to adjust the polling interval as needed.
+
+#### Improvements in the packages view
+
+Several improvements have been made to the packages view:
+* When deploying a blueprint or installing a plugin, users are now informed about the license that needs to be approved before proceeding further.
+* The extensions view has been improved with additional information about package types, offering better transparency about their origin and trustworthiness:
```

Comment:
What is the extensions view? Probably the last sentence must be moved to the top? Please add a screenshot here and a link to the user documentation where this is described.